Automattic/jetpack
# Social: Added Social specific success screen on activating Social plans.

## Proposed changes:

* Added Social-specific success screen. 
* Added redirection on the success screen's CTA based on whether the Jetpack plugin is active or if just the Social plugin is active. 

### Other information:


## Jetpack product discussion


pdrWKz-1nM-p2

## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* Buy the Jetpack Social Advanced plan, but don't attach it to a site. Go to https://cloud.jetpack.com/pricing/ to do so. 
* Then, head to `wp-admin/admin.php?page=my-jetpack#/add-license` to activate the Jetpack Social license you just bought. 
* See the new welcome screen as shown in the screengrab below. 
* Clicking on configure my site should take you to the recommendations screen if the Jetpack plugin is active. If the Jetpack Social plugin alone is active, you will be redirected to the Jetpack Social plugin's admin page. 
*  Buy the Jetpack Social basic plan and repeat the steps. You will want to remove Jetpack Social Advanced from your site if you want to try this on the same site. 
* The recommendation screens on the Jetpack plugin are being worked on in https://github.com/Automattic/jetpack/pull/35244, so they may not work correctly till the linked PR is merged. 


https://github.com/Automattic/jetpack/assets/6594561/7df163f2-3861-4ab1-b7c4-5696b2c3b59d
